# LinkML Registry

For more information on LinkML:

* [LinkML Documentation](https://linkml.io)
* [LinkML Meta Model Specification](https://w3id.org/linkml/docs/specification/)

## Registry Entries

| Title | Topics | Stars | GitHub URL |
|-------|--------|-------|-----------|
| [ontogpt](https://monarch-initiative.github.io/ontogpt/)<br/>LLM-based ontological extraction tools, including SPIRES |  | 701 [(details)](details/monarch-initiative_ontogpt.md) | [monarch-initiative/ontogpt](https://github.com/monarch-initiative/ontogpt) |
| [semantic-python-overview](https://pysemtec.github.io/semantic-python-overview/)<br/>(subjective) overview of projects which are related both to python and semantic technologies (RDF, O... |  | 524 [(details)](details/pysemtec_semantic-python-overview.md) | [pysemtec/semantic-python-overview](https://github.com/pysemtec/semantic-python-overview) |
| [awesome-ontology](https://ozekik.github.io/awesome-ontology/)<br/>A curated list of ontology things |  | 388 [(details)](details/ozekik_awesome-ontology.md) | [ozekik/awesome-ontology](https://github.com/ozekik/awesome-ontology) |
| [ontology-development-kit](https://incatools.github.io/ontology-development-kit/)<br/>Ontology lifecycle management |  | 260 [(details)](details/INCATools_ontology-development-kit.md) | [INCATools/ontology-development-kit](https://github.com/INCATools/ontology-development-kit) |
| [mondo](https://monarch-initiative.github.io/mondo/)<br/>Mondo Disease Ontology |  | 258 [(details)](details/monarch-initiative_mondo.md) | [monarch-initiative/mondo](https://github.com/monarch-initiative/mondo) |
| [ITK-Wasm](https://insightsoftwareconsortium.github.io/ITK-Wasm/)<br/>High performance spatial analysis in a web browser and across programming languages and hardware arc... |  | 215 [(details)](details/InsightSoftwareConsortium_ITK-Wasm.md) | [InsightSoftwareConsortium/ITK-Wasm](https://github.com/InsightSoftwareConsortium/ITK-Wasm) |
| [biolink-model](https://biolink.github.io/biolink-model/)<br/>Schema and generated objects for biolink data model and upper ontology | biolink, datamodel, gene-ontology... | 197 [(details)](details/biolink_biolink-model.md) | [biolink/biolink-model](https://github.com/biolink/biolink-model) |
| [sssom](https://mapping-commons.github.io/sssom/)<br/>Simple Standard for Sharing Ontology Mappings | linkml, mapping, mapping-commons... | 167 [(details)](details/mapping-commons_sssom.md) | [mapping-commons/sssom](https://github.com/mapping-commons/sssom) |
| [kgx](https://biolink.github.io/kgx/)<br/>KGX is a Python library for exchanging Knowledge Graphs |  | 128 [(details)](details/biolink_kgx.md) | [biolink/kgx](https://github.com/biolink/kgx) |
| [DataHarmonizer](https://cidgoh.github.io/DataHarmonizer/)<br/>A standardized browser-based spreadsheet editor and validator that can be run offline and locally, a... | application, data, harmonization... | 111 [(details)](details/cidgoh_DataHarmonizer.md) | [cidgoh/DataHarmonizer](https://github.com/cidgoh/DataHarmonizer) |
| [obo-relations](https://oborel.github.io/obo-relations/)<br/>RO is an ontology of relations for use with biological ontologies |  | 95 [(details)](details/oborel_obo-relations.md) | [oborel/obo-relations](https://github.com/oborel/obo-relations) |
| [kg-covid-19](https://knowledge-graph-hub.github.io/kg-covid-19/)<br/>An instance of KG Hub to produce a knowledge graph for COVID-19 response. |  | 82 [(details)](details/Knowledge-Graph-Hub_kg-covid-19.md) | [Knowledge-Graph-Hub/kg-covid-19](https://github.com/Knowledge-Graph-Hub/kg-covid-19) |
| [psych-DS](https://psych-ds.github.io/psych-DS/)<br/>YAML-based schema model for the Psych-DS project and its tools |  | 81 [(details)](details/psych-ds_psych-DS.md) | [psych-ds/psych-DS](https://github.com/psych-ds/psych-DS) |
| [risk-atlas-nexus](https://ibm.github.io/risk-atlas-nexus/)<br/>Risk Atlas Nexus: tooling to bring together resources related to governance of foundation models. |  | 66 [(details)](details/IBM_risk-atlas-nexus.md) | [IBM/risk-atlas-nexus](https://github.com/IBM/risk-atlas-nexus) |
| [CoreMS](https://emsl-computing.github.io/CoreMS/)<br/>CoreMS is a comprehensive mass spectrometry software framework |  | 59 [(details)](details/EMSL-Computing_CoreMS.md) | [EMSL-Computing/CoreMS](https://github.com/EMSL-Computing/CoreMS) |
| [sssom-py](https://mapping-commons.github.io/sssom-py/index.html#)<br/>Python toolkit for SSSOM mapping format | linkml, mappings, obofoundry... | 54 [(details)](details/mapping-commons_sssom-py.md) | [mapping-commons/sssom-py](https://github.com/mapping-commons/sssom-py) |
| [koza](https://monarch-initiative.github.io/koza/)<br/>Data transformation framework for LinkML data models | etl, knowledge-graph, koza... | 53 [(details)](details/monarch-initiative_koza.md) | [monarch-initiative/koza](https://github.com/monarch-initiative/koza) |
| [hdmf](https://hdmf-dev.github.io/hdmf/)<br/>The Hierarchical Data Modeling Framework |  | 50 [(details)](details/hdmf-dev_hdmf.md) | [hdmf-dev/hdmf](https://github.com/hdmf-dev/hdmf) |
| [go-site](https://geneontology.github.io/go-site/)<br/>A collection of metadata, tools, and files associated with the Gene Ontology public web presence. |  | 46 [(details)](details/geneontology_go-site.md) | [geneontology/go-site](https://github.com/geneontology/go-site) |
| [mixs](https://genomicsstandardsconsortium.github.io/mixs/)<br/>Minimum Information about any (X) Sequence‚Äù (MIxS) specification |  | 46 [(details)](details/GenomicsStandardsConsortium_mixs.md) | [GenomicsStandardsConsortium/mixs](https://github.com/GenomicsStandardsConsortium/mixs) |
| [semantic-sql](https://incatools.github.io/semantic-sql/)<br/>SQL and SQLite builds of OWL ontologies | linkml, oaklib, obofoundry... | 46 [(details)](details/INCATools_semantic-sql.md) | [INCATools/semantic-sql](https://github.com/INCATools/semantic-sql) |
| [specification](https://margo.github.io/specification/)<br/>Margo Specification |  | 39 [(details)](details/margo_specification.md) | [margo/specification](https://github.com/margo/specification) |
| [artificial-intelligence-ontology](https://berkeleybop.github.io/artificial-intelligence-ontology/)<br/>An ontology modeling classes and relationships describing deep learning networks, their component la... |  | 34 [(details)](details/berkeleybop_artificial-intelligence-ontology.md) | [berkeleybop/artificial-intelligence-ontology](https://github.com/berkeleybop/artificial-intelligence-ontology) |
| [nmdc-schema](https://microbiomedata.github.io/nmdc-schema/)<br/>National Microbiome Data Collaborative (NMDC) unified data model | ecoinformatics, fair-data, genomics... | 33 [(details)](details/microbiomedata_nmdc-schema.md) | [microbiomedata/nmdc-schema](https://github.com/microbiomedata/nmdc-schema) |
| [enola](https://enola-dev.github.io/enola/)<br/>Enola üïµüèæ‚Äç‚ôÄÔ∏è Holmes was an SRE. |  | 31 [(details)](details/enola-dev_enola.md) | [enola-dev/enola](https://github.com/enola-dev/enola) |
| [semantic-llama](https://cmungall.github.io/semantic-llama/)<br/>A knowledge extraction tool that uses a large language model to extract semantic information from te... | ai, knowledge-extraction, language-models... | 28 [(details)](details/cmungall_semantic-llama.md) | [cmungall/semantic-llama](https://github.com/cmungall/semantic-llama) |
| [nwb_hackathons](https://neurodatawithoutborders.github.io/nwb_hackathons/)<br/>Repository for collecting materials from nwb hackathons |  | 24 [(details)](details/NeurodataWithoutBorders_nwb_hackathons.md) | [NeurodataWithoutBorders/nwb_hackathons](https://github.com/NeurodataWithoutBorders/nwb_hackathons) |
| [RNA-KG](https://anacletolab.github.io/RNA-KG/) |  | 23 [(details)](details/AnacletoLAB_RNA-KG.md) | [AnacletoLAB/RNA-KG](https://github.com/AnacletoLAB/RNA-KG) |
| [cogs](https://colectica.github.io/cogs/)<br/>Convention-based Ontology Generation System |  | 20 [(details)](details/Colectica_cogs.md) | [Colectica/cogs](https://github.com/Colectica/cogs) |
| [kgcl](https://w3id.org/kgcl/)<br/>Datamodel for KGCL (Knowledge Graph Change Language) | kgcl, linkml, ontology-changes... | 18 [(details)](details/INCATools_kgcl.md) | [INCATools/kgcl](https://github.com/INCATools/kgcl) |
| [kgcl-rdflib](https://incatools.github.io/kgcl-rdflib/)<br/>Tools for working with KGCL | diff, diffs, kg... | 18 [(details)](details/INCATools_kgcl-rdflib.md) | [INCATools/kgcl-rdflib](https://github.com/INCATools/kgcl-rdflib) |
| [monarch-ingest](https://monarch-initiative.github.io/monarch-ingest/)<br/>Data ingest application for Monarch Initiative knowledge graph using Koza | kgx, knowledge-graphs, linkml... | 17 [(details)](details/monarch-initiative_monarch-ingest.md) | [monarch-initiative/monarch-ingest](https://github.com/monarch-initiative/monarch-ingest) |
| [ccdhmodel](https://cancerdhc.github.io/ccdhmodel/)<br/>CRDC-H model in LinkML, developed by the Center for Cancer Data Harmonization (CCDH) | google-sheets, linkml | 16 [(details)](details/cancerDHC_ccdhmodel.md) | [cancerDHC/ccdhmodel](https://github.com/cancerDHC/ccdhmodel) |
| [openstudybuilder-solution](https://novonordisk-opensource.github.io/openstudybuilder-solution/) |  | 15 [(details)](details/NovoNordisk-OpenSource_openstudybuilder-solution.md) | [NovoNordisk-OpenSource/openstudybuilder-solution](https://github.com/NovoNordisk-OpenSource/openstudybuilder-solution) |
| [ddimodel](https://ddialliance.github.io/ddimodel/)<br/>Model for DDI Lifecycle |  | 14 [(details)](details/ddialliance_ddimodel.md) | [ddialliance/ddimodel](https://github.com/ddialliance/ddimodel) |
| [ghga-metadata-schema](https://ghga-de.github.io/ghga-metadata-schema/)<br/>Metadata schema for the German Human Genome-Phenome Archive (GHGA) | fega, ghga, linkml... | 14 [(details)](details/ghga-de_ghga-metadata-schema.md) | [ghga-de/ghga-metadata-schema](https://github.com/ghga-de/ghga-metadata-schema) |
| [ukgov-metadata-exchange-model](https://co-cddo.github.io/ukgov-metadata-exchange-model/)<br/>A metadata model for describing data assets for exchanging between UK government organisations. |  | 14 [(details)](details/co-cddo_ukgov-metadata-exchange-model.md) | [co-cddo/ukgov-metadata-exchange-model](https://github.com/co-cddo/ukgov-metadata-exchange-model) |
| [markdown-corpus](https://tcr.github.io/markdown-corpus/)<br/>Many markdown files taken from around github |  | 13 [(details)](details/tcr_markdown-corpus.md) | [tcr/markdown-corpus](https://github.com/tcr/markdown-corpus) |
| [monochrom](http://monarch-initiative.github.io/monochrom/)<br/>Standardized identifiers and OWL classes for chromosomes and chromosomal parts across species | chromosomes, etl, genomics... | 13 [(details)](details/monarch-initiative_monochrom.md) | [monarch-initiative/monochrom](https://github.com/monarch-initiative/monochrom) |
| [py-typedlogic](https://py-typedlogic.github.io)<br/>Logic plus python types | answer-set-programming, clif, clingo... | 12 [(details)](details/py-typedlogic_py-typedlogic.md) | [py-typedlogic/py-typedlogic](https://github.com/py-typedlogic/py-typedlogic) |
| [translator-openpredict](https://maastrichtu-ids.github.io/translator-openpredict/)<br/>üîÆüêç A package to help serve predictions of biomedical concepts associations as Translator Reasoner AP... |  | 12 [(details)](details/MaastrichtU-IDS_translator-openpredict.md) | [MaastrichtU-IDS/translator-openpredict](https://github.com/MaastrichtU-IDS/translator-openpredict) |
| [metadata](https://isamplesorg.github.io/metadata/)<br/>Collation of metadata examples and notes for the project | linkml, metadata, physical-specimen | 11 [(details)](details/isamplesorg_metadata.md) | [isamplesorg/metadata](https://github.com/isamplesorg/metadata) |
| [political-affairs-ech-group](https://swiss.github.io/political-affairs-ech-group/)<br/>The working repository to develop standards for the swiss political system. |  | 11 [(details)](details/swiss_political-affairs-ech-group.md) | [swiss/political-affairs-ech-group](https://github.com/swiss/political-affairs-ech-group) |
| [json-flattener](https://cmungall.github.io/json-flattener/)<br/>Python library for denormalizing nested dicts or json objects to tables and back | dataframes, denormalization, linkml... | 10 [(details)](details/cmungall_json-flattener.md) | [cmungall/json-flattener](https://github.com/cmungall/json-flattener) |
| [kg-registry](https://knowledge-graph-hub.github.io/kg-registry/)<br/>A simple registry for KGHub projects |  | 9 [(details)](details/Knowledge-Graph-Hub_kg-registry.md) | [Knowledge-Graph-Hub/kg-registry](https://github.com/Knowledge-Graph-Hub/kg-registry) |
| [Hestia](https://ohdsi.github.io/Hestia/)<br/>Hestia is an API for function calling on the OMOP CDM. |  | 8 [(details)](details/OHDSI_Hestia.md) | [OHDSI/Hestia](https://github.com/OHDSI/Hestia) |
| [im-tc57cim](https://netbeheer-nederland.github.io/docs/im-tc57cim/latest)<br/>The CIM information model represented as LinkML schemas. |  | 8 [(details)](details/Netbeheer-Nederland_im-tc57cim.md) | [Netbeheer-Nederland/im-tc57cim](https://github.com/Netbeheer-Nederland/im-tc57cim) |
| [linkml-phenopackets](https://cmungall.github.io/linkml-phenopackets/)<br/>EXPERIMENTAL rendering of phenopackets in linkml | human-phenotype-ontology, linkml, metadata... | 8 [(details)](details/cmungall_linkml-phenopackets.md) | [cmungall/linkml-phenopackets](https://github.com/cmungall/linkml-phenopackets) |
| [SemanticDataLink](https://gx4fm-base-x.github.io/SemanticDataLink/)<br/>The Semantic Data Link repository hosts a Streamlit-based web application designed for managing and ... |  | 8 [(details)](details/GX4FM-Base-X_SemanticDataLink.md) | [GX4FM-Base-X/SemanticDataLink](https://github.com/GX4FM-Base-X/SemanticDataLink) |
| [agr_curation_schema](https://alliance-genome.github.io/agr_curation_schema/)<br/>Schema repository for the Alliance of Genome Resources persistent data store |  | 7 [(details)](details/alliance-genome_agr_curation_schema.md) | [alliance-genome/agr_curation_schema](https://github.com/alliance-genome/agr_curation_schema) |
| [datalad-concepts](https://psychoinformatics-de.github.io/datalad-concepts/) |  | 7 [(details)](details/psychoinformatics-de_datalad-concepts.md) | [psychoinformatics-de/datalad-concepts](https://github.com/psychoinformatics-de/datalad-concepts) |
| [dcat-4C-ap](https://stroemphi.github.io/dcat-4C-ap/) |  | 7 [(details)](details/StroemPhi_dcat-4C-ap.md) | [StroemPhi/dcat-4C-ap](https://github.com/StroemPhi/dcat-4C-ap) |
| [kg-bioportal](https://ncbo.github.io/kg-bioportal/)<br/>BioPortal, as a Knowledge Graph |  | 7 [(details)](details/ncbo_kg-bioportal.md) | [ncbo/kg-bioportal](https://github.com/ncbo/kg-bioportal) |
| [monarch-documentation](https://monarch-initiative.github.io/monarch-documentation/)<br/>Technical documentation for all Monarch applications, packages, services and related projects. |  | 7 [(details)](details/monarch-initiative_monarch-documentation.md) | [monarch-initiative/monarch-documentation](https://github.com/monarch-initiative/monarch-documentation) |
| [mondo-ingest](https://monarch-initiative.github.io/mondo-ingest/)<br/>Coordinating the mondo-ingest with external sources |  | 7 [(details)](details/monarch-initiative_mondo-ingest.md) | [monarch-initiative/mondo-ingest](https://github.com/monarch-initiative/mondo-ingest) |
| [DCAT-ap_as_LinkML_template](https://hendrikborgelt.github.io/DCAT-ap_as_LinkML_template/) |  | 6 [(details)](details/HendrikBorgelt_DCAT-ap_as_LinkML_template.md) | [HendrikBorgelt/DCAT-ap_as_LinkML_template](https://github.com/HendrikBorgelt/DCAT-ap_as_LinkML_template) |
| [gff-schema](https://biodatamodels.github.io/gff-schema/)<br/>LinkML schema for GFF3, the Genome Feature Format | bioinformatics, formats, genomics... | 6 [(details)](details/biodatamodels_gff-schema.md) | [biodatamodels/gff-schema](https://github.com/biodatamodels/gff-schema) |
| [data-vocab](https://govuk-one-login.github.io/data-vocab/)<br/>Vocabulary and schemas for digital identity |  | 5 [(details)](details/govuk-one-login_data-vocab.md) | [govuk-one-login/data-vocab](https://github.com/govuk-one-login/data-vocab) |
| [knowledge-graph-hub.github.io](https://knowledge-graph-hub.github.io/knowledge-graph-hub.github.io/) |  | 5 [(details)](details/Knowledge-Graph-Hub_knowledge-graph-hub.github.io.md) | [Knowledge-Graph-Hub/knowledge-graph-hub.github.io](https://github.com/Knowledge-Graph-Hub/knowledge-graph-hub.github.io) |
| [sample-annotator](https://microbiomedata.github.io/sample-annotator/static/intro.html)<br/>NMDC Sample Annotator | annotator, environment-ontology, environmentontology... | 5 [(details)](details/microbiomedata_sample-annotator.md) | [microbiomedata/sample-annotator](https://github.com/microbiomedata/sample-annotator) |
| [threesixtyeye](https://swhume.github.io/threesixtyeye/)<br/>Sam Hume's public 360i sandbox |  | 5 [(details)](details/swhume_threesixtyeye.md) | [swhume/threesixtyeye](https://github.com/swhume/threesixtyeye) |
| [ConfIDent_schema](https://tibhannover.github.io/ConfIDent_schema/)<br/>The data schema for the ConfIDent project. |  | 4 [(details)](details/TIBHannover_ConfIDent_schema.md) | [TIBHannover/ConfIDent_schema](https://github.com/TIBHannover/ConfIDent_schema) |
| [data-sheets-schema](https://bridge2ai.github.io/data-sheets-schema/)<br/>Datasheets for Datasets, as LinkML Schema |  | 4 [(details)](details/bridge2ai_data-sheets-schema.md) | [bridge2ai/data-sheets-schema](https://github.com/bridge2ai/data-sheets-schema) |
| [data-utilities](https://saved-models.github.io/data-utilities/)<br/>SAVED project - data processing tools for fish farms |  | 4 [(details)](details/saved-models_data-utilities.md) | [saved-models/data-utilities](https://github.com/saved-models/data-utilities) |
| [gen-linkml-profile](https://netbeheer-nederland.github.io/gen-linkml-profile/)<br/>LinkML schema profiler. |  | 4 [(details)](details/Netbeheer-Nederland_gen-linkml-profile.md) | [Netbeheer-Nederland/gen-linkml-profile](https://github.com/Netbeheer-Nederland/gen-linkml-profile) |
| [im-cgmes](https://netbeheer-nederland.github.io/im-cgmes/)<br/>ENTSO-E CGMES profiles represented as LinkML schemas. |  | 4 [(details)](details/Netbeheer-Nederland_im-cgmes.md) | [Netbeheer-Nederland/im-cgmes](https://github.com/Netbeheer-Nederland/im-cgmes) |
| [kgviz-model](https://berkeleybop.github.io/kgviz-model/)<br/>Schema for configurations / stylesheets for visualizing ontology and KGs | graphviz, javascript, kgs... | 4 [(details)](details/berkeleybop_kgviz-model.md) | [berkeleybop/kgviz-model](https://github.com/berkeleybop/kgviz-model) |
| [linkml-activitypub](https://p2p-ld.github.io/linkml-activitypub/)<br/>LinkML Schema representation of ActivityPub |  | 4 [(details)](details/p2p-ld_linkml-activitypub.md) | [p2p-ld/linkml-activitypub](https://github.com/p2p-ld/linkml-activitypub) |
| [mianct-schema](https://cmungall.github.io/mianct-schema/)<br/>Minimal Information About a New Cell Type | anatomy, cell-ontology, cell-types... | 4 [(details)](details/cmungall_mianct-schema.md) | [cmungall/mianct-schema](https://github.com/cmungall/mianct-schema) |
| [mtg_ontology](https://cmdoret.github.io/mtg_ontology/)<br/>An ontology describing cards in Magic The Gathering. | linkml, mtg, ontology... | 4 [(details)](details/cmdoret_mtg_ontology.md) | [cmdoret/mtg_ontology](https://github.com/cmdoret/mtg_ontology) |
| [nwb-linkml](https://nwb-linkml.readthedocs.io)<br/>Translating NWB schema language to linkml |  | 4 [(details)](details/p2p-ld_nwb-linkml.md) | [p2p-ld/nwb-linkml](https://github.com/p2p-ld/nwb-linkml) |
| [obograph](https://biodatamodels.github.io/obograph/) |  | 4 [(details)](details/biodatamodels_obograph.md) | [biodatamodels/obograph](https://github.com/biodatamodels/obograph) |
| [omnibenchmark](https://omnibenchmark.github.io/omnibenchmark/)<br/>The omnibenchmark CLI |  | 4 [(details)](details/omnibenchmark_omnibenchmark.md) | [omnibenchmark/omnibenchmark](https://github.com/omnibenchmark/omnibenchmark) |
| [ontology-associations](http://biodatamodels.github.io/ontology-associations)<br/>schema for ontology associations | annotation, bioinformatics, datamodels... | 4 [(details)](details/biodatamodels_ontology-associations.md) | [biodatamodels/ontology-associations](https://github.com/biodatamodels/ontology-associations) |
| [prefixcommons-py](https://prefixcommons.github.io/prefixcommons-py/)<br/>Prefix commons python utlities |  | 4 [(details)](details/prefixcommons_prefixcommons-py.md) | [prefixcommons/prefixcommons-py](https://github.com/prefixcommons/prefixcommons-py) |
| [rarelink](https://bih-cei.github.io/rarelink/)<br/>RareLink - A REDCap-based framework linking international registries toFHIR and Phenopackets |  | 4 [(details)](details/BIH-CEI_rarelink.md) | [BIH-CEI/rarelink](https://github.com/BIH-CEI/rarelink) |
| [cim-to-linkml](https://netbeheer-nederland.github.io/cim-to-linkml/)<br/>Generates LinkML schemas for packages in the CIM information model. |  | 3 [(details)](details/Netbeheer-Nederland_cim-to-linkml.md) | [Netbeheer-Nederland/cim-to-linkml](https://github.com/Netbeheer-Nederland/cim-to-linkml) |
| [eco-kg](https://knowledge-graph-hub.github.io/eco-kg/) |  | 3 [(details)](details/Knowledge-Graph-Hub_eco-kg.md) | [Knowledge-Graph-Hub/eco-kg](https://github.com/Knowledge-Graph-Hub/eco-kg) |
| [hra-do-processor](https://hubmapconsortium.github.io/hra-do-processor/)<br/>hra-do-processor: HRA Digital Objects Processor |  | 3 [(details)](details/hubmapconsortium_hra-do-processor.md) | [hubmapconsortium/hra-do-processor](https://github.com/hubmapconsortium/hra-do-processor) |
| [include-linkml](https://include-dcc.github.io/include-linkml/)<br/>LinkML Schema for INCLUDE DCC |  | 3 [(details)](details/include-dcc_include-linkml.md) | [include-dcc/include-linkml](https://github.com/include-dcc/include-linkml) |
| [information-resource-registry](https://biolink.github.io/information-resource-registry/)<br/>The information resource registry is a listing of data sources present in the NCATS Data Translator ... |  | 3 [(details)](details/biolink_information-resource-registry.md) | [biolink/information-resource-registry](https://github.com/biolink/information-resource-registry) |
| [KGQA-COVID](https://mrbearwithhissword.github.io/KGQA-COVID/) |  | 3 [(details)](details/MrBearWithHisSword_KGQA-COVID.md) | [MrBearWithHisSword/KGQA-COVID](https://github.com/MrBearWithHisSword/KGQA-COVID) |
| [mifc](https://kaiiam.github.io/mifc/)<br/>A minimum information standard checklist formalizing the description of food composition data and re... |  | 3 [(details)](details/kaiiam_mifc.md) | [kaiiam/mifc](https://github.com/kaiiam/mifc) |
| [miiid-schema](https://fair-mi.github.io/miiid-schema/)<br/>A metadata schema for the Minimum Information about Intermicrobial Interaction Data (MIIID) using Li... | fair-data, linkml, linkml-schema... | 3 [(details)](details/FAIR-MI_miiid-schema.md) | [FAIR-MI/miiid-schema](https://github.com/FAIR-MI/miiid-schema) |
| [model-card-schema](https://bridge2ai.github.io/model-card-schema/)<br/>LinkML rendering of model cards | ai, bridge2ai, deep-learning... | 3 [(details)](details/bridge2ai_model-card-schema.md) | [bridge2ai/model-card-schema](https://github.com/bridge2ai/model-card-schema) |
| [models](https://brain-bican.github.io/models/)<br/>BICAN data models |  | 3 [(details)](details/brain-bican_models.md) | [brain-bican/models](https://github.com/brain-bican/models) |
| [pid4cat-model](https://nfdi4cat.github.io/pid4cat-model/)<br/>Handle-based PIDs with a rich LinkML metadata model for resources in catalysis (PID4Cat) | handles, linkml, linkml-schema... | 3 [(details)](details/nfdi4cat_pid4cat-model.md) | [nfdi4cat/pid4cat-model](https://github.com/nfdi4cat/pid4cat-model) |
| [standards-schemas](https://bridge2ai.github.io/standards-schemas/)<br/>Data schema for Bridge2AI Standards. |  | 3 [(details)](details/bridge2ai_standards-schemas.md) | [bridge2ai/standards-schemas](https://github.com/bridge2ai/standards-schemas) |
| [TermCI-model](https://timsbiomed.github.io/TermCI-model/)<br/>The Terminology Code Index Entry model |  | 3 [(details)](details/timsbiomed_TermCI-model.md) | [timsbiomed/TermCI-model](https://github.com/timsbiomed/TermCI-model) |
| [wot-thing-description-toolchain-tmp](https://w3c.github.io/wot-thing-description-toolchain-tmp/)<br/>work area for WoT Thing Description toolchain |  | 3 [(details)](details/w3c_wot-thing-description-toolchain-tmp.md) | [w3c/wot-thing-description-toolchain-tmp](https://github.com/w3c/wot-thing-description-toolchain-tmp) |
| [alpha-repos](https://krishnapraveen7.github.io/alpha-repos/) |  | 2 [(details)](details/krishnapraveen7_alpha-repos.md) | [krishnapraveen7/alpha-repos](https://github.com/krishnapraveen7/alpha-repos) |
| [BDP-Ontology](https://hsolbrig.github.io/BDP-Ontology/) |  | 2 [(details)](details/hsolbrig_BDP-Ontology.md) | [hsolbrig/BDP-Ontology](https://github.com/hsolbrig/BDP-Ontology) |
| [bibliomancer](https://cmungall.github.io/bibliomancer/)<br/>Performs magic on bibliographies |  | 2 [(details)](details/cmungall_bibliomancer.md) | [cmungall/bibliomancer](https://github.com/cmungall/bibliomancer) |
| [brc-schema](https://bioenergy-research-centers.github.io/brc-schema/)<br/>LinkML schema repo for bioenergy.org data integration | linkml-schema | 2 [(details)](details/bioenergy-research-centers_brc-schema.md) | [bioenergy-research-centers/brc-schema](https://github.com/bioenergy-research-centers/brc-schema) |
| [CIM4Diagram](https://sveino.github.io/CIM4Diagram/)<br/>CIM use of LinkML for creating vocabulary and profile for Diagram |  | 2 [(details)](details/Sveino_CIM4Diagram.md) | [Sveino/CIM4Diagram](https://github.com/Sveino/CIM4Diagram) |
| [DataExchange-ODM-LinkML](https://cdisc-org.github.io/DataExchange-ODM-LinkML/)<br/>LinkML related work for ODMv2 | data-exchange-standard | 2 [(details)](details/cdisc-org_DataExchange-ODM-LinkML.md) | [cdisc-org/DataExchange-ODM-LinkML](https://github.com/cdisc-org/DataExchange-ODM-LinkML) |
| [DEPRECATED_agr_persistent_schema](https://alliance-genome.github.io/DEPRECATED_agr_persistent_schema/)<br/>Repository using linkml templating system for generating Alliance of Genome Resources persistent dat... | linkml | 2 [(details)](details/alliance-genome_DEPRECATED_agr_persistent_schema.md) | [alliance-genome/DEPRECATED_agr_persistent_schema](https://github.com/alliance-genome/DEPRECATED_agr_persistent_schema) |
| [Explanatory-Agent](https://ncatstranslator.github.io/Explanatory-Agent/) |  | 2 [(details)](details/NCATSTranslator_Explanatory-Agent.md) | [NCATSTranslator/Explanatory-Agent](https://github.com/NCATSTranslator/Explanatory-Agent) |
| [gaganyatri.in](https://slabstech.github.io/gaganyatri.in/)<br/>Gaganyatri - Space Operations for Mars Transfer |  | 2 [(details)](details/slabstech_gaganyatri.in.md) | [slabstech/gaganyatri.in](https://github.com/slabstech/gaganyatri.in) |
| [irud_keio](https://bih-cei.github.io/irud_keio/) |  | 2 [(details)](details/BIH-CEI_irud_keio.md) | [BIH-CEI/irud_keio](https://github.com/BIH-CEI/irud_keio) |
| [mixs-source](https://cmungall.github.io/mixs-source/) |  | 2 [(details)](details/cmungall_mixs-source.md) | [cmungall/mixs-source](https://github.com/cmungall/mixs-source) |
| [monarch-provenance](https://matentzn.github.io/monarch-provenance/) |  | 2 [(details)](details/matentzn_monarch-provenance.md) | [matentzn/monarch-provenance](https://github.com/matentzn/monarch-provenance) |
| [neat-ml-schema](https://knowledge-graph-hub.github.io/neat-ml-schema/)<br/>A repository that contains the LinkML model for specifying graph ML pipelines |  | 2 [(details)](details/Knowledge-Graph-Hub_neat-ml-schema.md) | [Knowledge-Graph-Hub/neat-ml-schema](https://github.com/Knowledge-Graph-Hub/neat-ml-schema) |
| [OpenCloning_LinkML](https://opencloning.github.io/OpenCloning_LinkML/)<br/>A LinkML data model for OpenCloning |  | 2 [(details)](details/OpenCloning_OpenCloning_LinkML.md) | [OpenCloning/OpenCloning_LinkML](https://github.com/OpenCloning/OpenCloning_LinkML) |
| [PPODLinkML](https://adhollander.github.io/PPODLinkML/)<br/>A LinkML representation of the PPOD schema |  | 2 [(details)](details/adhollander_PPODLinkML.md) | [adhollander/PPODLinkML](https://github.com/adhollander/PPODLinkML) |
| [sdk-adlib-exporter](https://av-efi.github.io/sdk-adlib-exporter/)<br/>Code used at Stiftung Deutsche Kinemathek to register AVefi compliant PIDs for collections on record... |  | 2 [(details)](details/AV-EFI_sdk-adlib-exporter.md) | [AV-EFI/sdk-adlib-exporter](https://github.com/AV-EFI/sdk-adlib-exporter) |
| [sepio-linkml](https://sepio-framework.github.io/sepio-linkml/)<br/>LinkML-based specification of the SEPIO information model. |  | 2 [(details)](details/sepio-framework_sepio-linkml.md) | [sepio-framework/sepio-linkml](https://github.com/sepio-framework/sepio-linkml) |
| [valve-linkml](https://tmprd.github.io/valve-linkml/)<br/>Convert LinkML schema + data to VALVE tables and vice versa |  | 2 [(details)](details/tmprd_valve-linkml.md) | [tmprd/valve-linkml](https://github.com/tmprd/valve-linkml) |
| [zebrafish-toxicology-atlas-schema](https://ehs-data-standards.github.io/zebrafish-toxicology-atlas-schema/) |  | 2 [(details)](details/EHS-Data-Standards_zebrafish-toxicology-atlas-schema.md) | [EHS-Data-Standards/zebrafish-toxicology-atlas-schema](https://github.com/EHS-Data-Standards/zebrafish-toxicology-atlas-schema) |
| [bdfkb-schema](https://arpa-h-bdf.github.io/bdfkb-schema/) |  | 1 [(details)](details/ARPA-H-BDF_bdfkb-schema.md) | [ARPA-H-BDF/bdfkb-schema](https://github.com/ARPA-H-BDF/bdfkb-schema) |
| [cas-tools](https://cellannotation.github.io/cas-tools/)<br/>Cell Annotation Schema Tools |  | 1 [(details)](details/cellannotation_cas-tools.md) | [cellannotation/cas-tools](https://github.com/cellannotation/cas-tools) |
| [comet](https://cmungall.github.io/comet/) |  | 1 [(details)](details/cmungall_comet.md) | [cmungall/comet](https://github.com/cmungall/comet) |
| [gocam-py](https://geneontology.github.io/gocam-py/)<br/>Draft data model for GO-CAMs |  | 1 [(details)](details/geneontology_gocam-py.md) | [geneontology/gocam-py](https://github.com/geneontology/gocam-py) |
| [knowledge-graph](https://genophenoenvo.github.io/knowledge-graph/) |  | 1 [(details)](details/genophenoenvo_knowledge-graph.md) | [genophenoenvo/knowledge-graph](https://github.com/genophenoenvo/knowledge-graph) |
| [linkml-aind-model](https://dailydreaming.github.io/linkml-aind-model/)<br/>Translate pydantic-based schemas from python modules to LinkML. |  | 1 [(details)](details/DailyDreaming_linkml-aind-model.md) | [DailyDreaming/linkml-aind-model](https://github.com/DailyDreaming/linkml-aind-model) |
| [linkml-data-dictionary](https://chicagopcdc.github.io/linkml-data-dictionary/)<br/>Define the data dictionary using LinkML for data validation and reusable artifacts. (Proof of concep... |  | 1 [(details)](details/chicagopcdc_linkml-data-dictionary.md) | [chicagopcdc/linkml-data-dictionary](https://github.com/chicagopcdc/linkml-data-dictionary) |
| [linkml-project-copier](https://dalito.github.io/linkml-project-copier/)<br/>A copier template for your next LinkML project |  | 1 [(details)](details/dalito_linkml-project-copier.md) | [dalito/linkml-project-copier](https://github.com/dalito/linkml-project-copier) |
| [linkml_tools](https://kgerring.github.io/linkml_tools/)<br/>grouping pf useful others |  | 1 [(details)](details/KGerring_linkml_tools.md) | [KGerring/linkml_tools](https://github.com/KGerring/linkml_tools) |
| [mofdscribe-schema](https://kjappelbaum.github.io/mofdscribe-schema/) |  | 1 [(details)](details/kjappelbaum_mofdscribe-schema.md) | [kjappelbaum/mofdscribe-schema](https://github.com/kjappelbaum/mofdscribe-schema) |
| [nmdc-client](https://jeffbaumes.github.io/nmdc-client/)<br/>A Python client for the National Microbiome Data Collaborative |  | 1 [(details)](details/jeffbaumes_nmdc-client.md) | [jeffbaumes/nmdc-client](https://github.com/jeffbaumes/nmdc-client) |
| [NMRspec](https://stroemphi.github.io/NMRspec/)<br/>Prototype to semantify NMR spectroscopy research data |  | 1 [(details)](details/StroemPhi_NMRspec.md) | [StroemPhi/NMRspec](https://github.com/StroemPhi/NMRspec) |
| [sheet2linkml](https://cancerdhc.github.io/sheet2linkml/)<br/>Python package for conversion of Google Sheet to LinkML for CCDH |  | 1 [(details)](details/cancerDHC_sheet2linkml.md) | [cancerDHC/sheet2linkml](https://github.com/cancerDHC/sheet2linkml) |
| [SketchML](https://axenov.github.io/SketchML/)<br/>:whale: SketchML - distributed Machine Leaning framework for Apache Flink :star: |  | 1 [(details)](details/axenov_SketchML.md) | [axenov/SketchML](https://github.com/axenov/SketchML) |
| [smpdb-beacon](https://ncats-tangerine.github.io/smpdb-beacon/) |  | 1 [(details)](details/NCATS-Tangerine_smpdb-beacon.md) | [NCATS-Tangerine/smpdb-beacon](https://github.com/NCATS-Tangerine/smpdb-beacon) |
| [structsense](https://sensein.github.io/structsense/) |  | 1 [(details)](details/sensein_structsense.md) | [sensein/structsense](https://github.com/sensein/structsense) |
| [tccm-model-lite](https://timsbiomed.github.io/tccm-model-lite/) |  | 1 [(details)](details/timsbiomed_tccm-model-lite.md) | [timsbiomed/tccm-model-lite](https://github.com/timsbiomed/tccm-model-lite) |
| [ai_data](https://bioimage-archive.github.io/ai_data/) |  | 0 [(details)](details/BioImage-Archive_ai_data.md) | [BioImage-Archive/ai_data](https://github.com/BioImage-Archive/ai_data) |
| [arcZcode](https://sveino.github.io/arcZcode/)<br/>Architecture as Code |  | 0 [(details)](details/Sveino_arcZcode.md) | [Sveino/arcZcode](https://github.com/Sveino/arcZcode) |
| [automate_mappings](https://sierra-moxon.github.io/automate_mappings/) |  | 0 [(details)](details/sierra-moxon_automate_mappings.md) | [sierra-moxon/automate_mappings](https://github.com/sierra-moxon/automate_mappings) |
| [basin3d_schema](https://cmungall.github.io/basin3d_schema/)<br/>EXPERIMENTAL |  | 0 [(details)](details/cmungall_basin3d_schema.md) | [cmungall/basin3d_schema](https://github.com/cmungall/basin3d_schema) |
| [biotic-interactions](https://diatomsrcool.github.io/biotic-interactions/)<br/>LinkML model for biotic interactions - designed for use with OntoGPT |  | 0 [(details)](details/diatomsRcool_biotic-interactions.md) | [diatomsRcool/biotic-interactions](https://github.com/diatomsRcool/biotic-interactions) |
| [cell-annotation-schema](https://cellular-semantics.github.io/cell-annotation-schema/) |  | 0 [(details)](details/Cellular-Semantics_cell-annotation-schema.md) | [Cellular-Semantics/cell-annotation-schema](https://github.com/Cellular-Semantics/cell-annotation-schema) |
| [cellmark_schema](https://cellular-semantics.github.io/cellmark_schema/) |  | 0 [(details)](details/Cellular-Semantics_cellmark_schema.md) | [Cellular-Semantics/cellmark_schema](https://github.com/Cellular-Semantics/cellmark_schema) |
| [clinical-studies-schema](https://sierra-moxon.github.io/clinical-studies-schema/) |  | 0 [(details)](details/sierra-moxon_clinical-studies-schema.md) | [sierra-moxon/clinical-studies-schema](https://github.com/sierra-moxon/clinical-studies-schema) |
| [CPKG](https://ncats-tangerine.github.io/CPKG/)<br/>Translation tools for ClinicalProfile to Knowledge Graphs |  | 0 [(details)](details/NCATS-Tangerine_CPKG.md) | [NCATS-Tangerine/CPKG](https://github.com/NCATS-Tangerine/CPKG) |
| [dandisets-linkml-status-tools](https://dandi.github.io/dandisets-linkml-status-tools/)<br/>Tools for reporting the status of the translation of DANDI schemas from Pydantic definitions to Link... |  | 0 [(details)](details/dandi_dandisets-linkml-status-tools.md) | [dandi/dandisets-linkml-status-tools](https://github.com/dandi/dandisets-linkml-status-tools) |
| [DataMetadataWorkflowDocs](https://in2primatebrains.github.io/DataMetadataWorkflowDocs/)<br/>Documentation for In2PrimateBrains Data and Metadata Workflows |  | 0 [(details)](details/In2PrimateBrains_DataMetadataWorkflowDocs.md) | [In2PrimateBrains/DataMetadataWorkflowDocs](https://github.com/In2PrimateBrains/DataMetadataWorkflowDocs) |
| [dp-meetdata](https://netbeheer-nederland.github.io/dp-meetdata/)<br/>Information models for the Meetdata data product. |  | 0 [(details)](details/Netbeheer-Nederland_dp-meetdata.md) | [Netbeheer-Nederland/dp-meetdata](https://github.com/Netbeheer-Nederland/dp-meetdata) |
| [edd-schema](https://cmungall.github.io/edd-schema/) |  | 0 [(details)](details/cmungall_edd-schema.md) | [cmungall/edd-schema](https://github.com/cmungall/edd-schema) |
| [external-metadata-awareness](https://microbiomedata.github.io/external-metadata-awareness/)<br/>Tools for fetching and processing external schemas, ontologies and metadata records. Includes value ... |  | 0 [(details)](details/microbiomedata_external-metadata-awareness.md) | [microbiomedata/external-metadata-awareness](https://github.com/microbiomedata/external-metadata-awareness) |
| [foo-model](https://hsolbrig.github.io/foo-model/) |  | 0 [(details)](details/hsolbrig_foo-model.md) | [hsolbrig/foo-model](https://github.com/hsolbrig/foo-model) |
| [ga4gh-va-schema](https://sujaypatil96.github.io/ga4gh-va-schema/)<br/>Data model for variant annotation in the GA4GH consortium. |  | 0 [(details)](details/sujaypatil96_ga4gh-va-schema.md) | [sujaypatil96/ga4gh-va-schema](https://github.com/sujaypatil96/ga4gh-va-schema) |
| [ghp-vocab](https://mukundanps.github.io/ghp-vocab/)<br/>Vocabulary for Good Health Pass |  | 0 [(details)](details/mukundanps_ghp-vocab.md) | [mukundanps/ghp-vocab](https://github.com/mukundanps/ghp-vocab) |
| [github-helperbot](https://tallboy17.github.io/github-helperbot/)<br/>AI Agent to research and search for projects on GitHub |  | 0 [(details)](details/tallboy17_github-helperbot.md) | [tallboy17/github-helperbot](https://github.com/tallboy17/github-helperbot) |
| [hca-validation-tools](https://clevercanary.github.io/hca-validation-tools/) |  | 0 [(details)](details/clevercanary_hca-validation-tools.md) | [clevercanary/hca-validation-tools](https://github.com/clevercanary/hca-validation-tools) |
| [hdmf-term-lists](https://rly.github.io/hdmf-term-lists/) |  | 0 [(details)](details/rly_hdmf-term-lists.md) | [rly/hdmf-term-lists](https://github.com/rly/hdmf-term-lists) |
| [idr_metadata_model](https://khaledk2.github.io/idr_metadata_model/) |  | 0 [(details)](details/khaledk2_idr_metadata_model.md) | [khaledk2/idr_metadata_model](https://github.com/khaledk2/idr_metadata_model) |
| [in2pb-linkml-test](https://ree-gupta.github.io/in2pb-linkml-test/) |  | 0 [(details)](details/ree-gupta_in2pb-linkml-test.md) | [ree-gupta/in2pb-linkml-test](https://github.com/ree-gupta/in2pb-linkml-test) |
| [jsonasobj](https://hsolbrig.github.io/jsonasobj/)<br/>Extension to the python json library that represents JSON as first class python objects. |  | 0 [(details)](details/hsolbrig_jsonasobj.md) | [hsolbrig/jsonasobj](https://github.com/hsolbrig/jsonasobj) |
| [kg-envpolyreg](https://knowledge-graph-hub.github.io/kg-envpolyreg/) |  | 0 [(details)](details/Knowledge-Graph-Hub_kg-envpolyreg.md) | [Knowledge-Graph-Hub/kg-envpolyreg](https://github.com/Knowledge-Graph-Hub/kg-envpolyreg) |
| [linkml-cfde](https://cmungall.github.io/linkml-cfde/)<br/>EXPERIMENTAL rendering of Common Fund Data Ecocystem as LinkML |  | 0 [(details)](details/cmungall_linkml-cfde.md) | [cmungall/linkml-cfde](https://github.com/cmungall/linkml-cfde) |
| [linkml-enhanced-template](https://hsolbrig.github.io/linkml-enhanced-template/)<br/>Linkml template playground |  | 0 [(details)](details/hsolbrig_linkml-enhanced-template.md) | [hsolbrig/linkml-enhanced-template](https://github.com/hsolbrig/linkml-enhanced-template) |
| [linkml-examples](https://rsc3.github.io/linkml-examples/)<br/>Tests and examples of linkml |  | 0 [(details)](details/rsc3_linkml-examples.md) | [rsc3/linkml-examples](https://github.com/rsc3/linkml-examples) |
| [linkml-gen-something-functional](https://turbomam.github.io/linkml-gen-something-functional/)<br/>Fragments of schemas including features that may not work with all genrated artifacts |  | 0 [(details)](details/turbomam_linkml-gen-something-functional.md) | [turbomam/linkml-gen-something-functional](https://github.com/turbomam/linkml-gen-something-functional) |
| [linkml-playground](https://manulera.github.io/linkml-playground/)<br/>linkml-playground |  | 0 [(details)](details/manulera_linkml-playground.md) | [manulera/linkml-playground](https://github.com/manulera/linkml-playground) |
| [linkml-pokemon](https://vladistan.github.io/linkml-pokemon/) |  | 0 [(details)](details/vladistan_linkml-pokemon.md) | [vladistan/linkml-pokemon](https://github.com/vladistan/linkml-pokemon) |
| [linkml-toolkit](https://genomewalker.github.io/linkml-toolkit/) |  | 0 [(details)](details/genomewalker_linkml-toolkit.md) | [genomewalker/linkml-toolkit](https://github.com/genomewalker/linkml-toolkit) |
| [linkml-tutorial-2024](https://hoodriverheather.github.io/linkml-tutorial-2024/) |  | 0 [(details)](details/hoodriverheather_linkml-tutorial-2024.md) | [hoodriverheather/linkml-tutorial-2024](https://github.com/hoodriverheather/linkml-tutorial-2024) |
| [linkml-tutorial-2024](https://sumirp.github.io/linkml-tutorial-2024/) |  | 0 [(details)](details/sumirp_linkml-tutorial-2024.md) | [sumirp/linkml-tutorial-2024](https://github.com/sumirp/linkml-tutorial-2024) |
| [linkml-tutorial-2024](https://cwseidel.github.io/linkml-tutorial-2024/)<br/>LinkML tutorial from ISMB 2024 |  | 0 [(details)](details/cwseidel_linkml-tutorial-2024.md) | [cwseidel/linkml-tutorial-2024](https://github.com/cwseidel/linkml-tutorial-2024) |
| [matrix-schema](https://everycure-org.github.io/matrix-schema/) |  | 0 [(details)](details/everycure-org_matrix-schema.md) | [everycure-org/matrix-schema](https://github.com/everycure-org/matrix-schema) |
| [metadata_converter](https://microbiomedata.github.io/metadata_converter/)<br/>Conversion of schemas/templates/metadata to biolinkml |  | 0 [(details)](details/microbiomedata_metadata_converter.md) | [microbiomedata/metadata_converter](https://github.com/microbiomedata/metadata_converter) |
| [microscopemetrics-schema](https://montpellierressourcesimagerie.github.io/microscopemetrics-schema/) |  | 0 [(details)](details/MontpellierRessourcesImagerie_microscopemetrics-schema.md) | [MontpellierRessourcesImagerie/microscopemetrics-schema](https://github.com/MontpellierRessourcesImagerie/microscopemetrics-schema) |
| [minimal-metab](https://microbiomedata.github.io/minimal-metab/) |  | 0 [(details)](details/microbiomedata_minimal-metab.md) | [microbiomedata/minimal-metab](https://github.com/microbiomedata/minimal-metab) |
| [modos-schema](https://sdsc-ordes.github.io/modos-schema/)<br/>Metadata schema for the SMOC Multi-Omics Digital-Object |  | 0 [(details)](details/sdsc-ordes_modos-schema.md) | [sdsc-ordes/modos-schema](https://github.com/sdsc-ordes/modos-schema) |
| [nmdc-metadata](https://microbiomedata.github.io/nmdc-metadata/)<br/>Managing metadata and policy around metadata in NMDC |  | 0 [(details)](details/microbiomedata_nmdc-metadata.md) | [microbiomedata/nmdc-metadata](https://github.com/microbiomedata/nmdc-metadata) |
| [nmdc-runtime](https://microbiomedata.github.io/nmdc-runtime/)<br/>Runtime system for NMDC data management and orchestration |  | 0 [(details)](details/microbiomedata_nmdc-runtime.md) | [microbiomedata/nmdc-runtime](https://github.com/microbiomedata/nmdc-runtime) |
| [obo-metadata](https://cmungall.github.io/obo-metadata/)<br/>EXPERIMENTAL obo metadata repo |  | 0 [(details)](details/cmungall_obo-metadata.md) | [cmungall/obo-metadata](https://github.com/cmungall/obo-metadata) |
| [omop_to_rdf](https://sierra-moxon.github.io/omop_to_rdf/)<br/>test repo for automating conversion from omop tables to rdf via linkml-schema-automator and linkml-c... |  | 0 [(details)](details/sierra-moxon_omop_to_rdf.md) | [sierra-moxon/omop_to_rdf](https://github.com/sierra-moxon/omop_to_rdf) |
| [Ontology-Metadata](https://stroemphi.github.io/Ontology-Metadata/)<br/>A schema for ontology metadata (WIP) |  | 0 [(details)](details/StroemPhi_Ontology-Metadata.md) | [StroemPhi/Ontology-Metadata](https://github.com/StroemPhi/Ontology-Metadata) |
| [ontology_to_schema](https://liuji1031.github.io/ontology_to_schema/) |  | 0 [(details)](details/liuji1031_ontology_to_schema.md) | [liuji1031/ontology_to_schema](https://github.com/liuji1031/ontology_to_schema) |
| [operando-catalysis-metadata-schema](https://anak-velazquez.github.io/operando-catalysis-metadata-schema/)<br/>[HZB] operando Catalysis (meta)data schema |  | 0 [(details)](details/anak-velazquez_operando-catalysis-metadata-schema.md) | [anak-velazquez/operando-catalysis-metadata-schema](https://github.com/anak-velazquez/operando-catalysis-metadata-schema) |
| [oregon-water-samples](https://sierra-moxon.github.io/oregon-water-samples/)<br/>Sample LinkML project setup for 2024 paper figures |  | 0 [(details)](details/sierra-moxon_oregon-water-samples.md) | [sierra-moxon/oregon-water-samples](https://github.com/sierra-moxon/oregon-water-samples) |
| [PCDMLinkML](https://multimeric.github.io/PCDMLinkML/)<br/>Portland Common Data Model, in LinkML |  | 0 [(details)](details/multimeric_PCDMLinkML.md) | [multimeric/PCDMLinkML](https://github.com/multimeric/PCDMLinkML) |
| [pfhub-cli](https://usnistgov.github.io/pfhub-cli/)<br/>PFHub CLI tool |  | 0 [(details)](details/usnistgov_pfhub-cli.md) | [usnistgov/pfhub-cli](https://github.com/usnistgov/pfhub-cli) |
| [pfhub-cli-old](https://wd15.github.io/pfhub-cli-old/) |  | 0 [(details)](details/wd15_pfhub-cli-old.md) | [wd15/pfhub-cli-old](https://github.com/wd15/pfhub-cli-old) |
| [portable-microhaplotype-object](https://plasmogenepi.github.io/portable-microhaplotype-object/)<br/>A schema to define the minimum amount of data needed to export a microhaplotype calling pipeline ana... |  | 0 [(details)](details/PlasmoGenEpi_portable-microhaplotype-object.md) | [PlasmoGenEpi/portable-microhaplotype-object](https://github.com/PlasmoGenEpi/portable-microhaplotype-object) |
| [psyndex2linkeddata](https://leibniz-psychology.github.io/psyndex2linkeddata/)<br/>A set of python tools for describing PSYNDEX data in RDF and for converting from STARXML to Linked D... |  | 0 [(details)](details/leibniz-psychology_psyndex2linkeddata.md) | [leibniz-psychology/psyndex2linkeddata](https://github.com/leibniz-psychology/psyndex2linkeddata) |
| [refscan](https://microbiomedata.github.io/refscan/)<br/>Command-line program that scans NMDC database for referential integrity violations |  | 0 [(details)](details/microbiomedata_refscan.md) | [microbiomedata/refscan](https://github.com/microbiomedata/refscan) |
| [research-paper-schema](https://liuji1031.github.io/research-paper-schema/) |  | 0 [(details)](details/liuji1031_research-paper-schema.md) | [liuji1031/research-paper-schema](https://github.com/liuji1031/research-paper-schema) |
| [rhea-beacon](https://ncats-tangerine.github.io/rhea-beacon/)<br/>Knowledge Beacon wrapper for the Rhea Annotated Reactions database (https://www.rhea-db.org) |  | 0 [(details)](details/NCATS-Tangerine_rhea-beacon.md) | [NCATS-Tangerine/rhea-beacon](https://github.com/NCATS-Tangerine/rhea-beacon) |
| [schema-automator-monarch](https://sierra-moxon.github.io/schema-automator-monarch/) |  | 0 [(details)](details/sierra-moxon_schema-automator-monarch.md) | [sierra-moxon/schema-automator-monarch](https://github.com/sierra-moxon/schema-automator-monarch) |
| [semanticscience-model](https://vemonet.github.io/semanticscience-model/)<br/>LinkML model for the SemanticScience Integrated Ontology |  | 0 [(details)](details/vemonet_semanticscience-model.md) | [vemonet/semanticscience-model](https://github.com/vemonet/semanticscience-model) |
| [sepio-models](https://clingen-data-model.github.io/sepio-models/)<br/>Shared SEPIO models based on the bioLinkML modeling language supporting the metakb and clingen sepio... |  | 0 [(details)](details/clingen-data-model_sepio-models.md) | [clingen-data-model/sepio-models](https://github.com/clingen-data-model/sepio-models) |
| [submission-schema](https://microbiomedata.github.io/submission-schema/) |  | 0 [(details)](details/microbiomedata_submission-schema.md) | [microbiomedata/submission-schema](https://github.com/microbiomedata/submission-schema) |
| [synbio-schema](https://iarpa-felix.github.io/synbio-schema/)<br/>A schema for synthetic biology, inspired by IARPA FELIX |  | 0 [(details)](details/iarpa-felix_synbio-schema.md) | [iarpa-felix/synbio-schema](https://github.com/iarpa-felix/synbio-schema) |
| [tank-schema](https://turbomam.github.io/tank-schema/) |  | 0 [(details)](details/turbomam_tank-schema.md) | [turbomam/tank-schema](https://github.com/turbomam/tank-schema) |
| [tccm-api](https://timsbiomed.github.io/tccm-api/) |  | 0 [(details)](details/timsbiomed_tccm-api.md) | [timsbiomed/tccm-api](https://github.com/timsbiomed/tccm-api) |
| [template-fun-model](https://hsolbrig.github.io/template-fun-model/)<br/>Demo |  | 0 [(details)](details/hsolbrig_template-fun-model.md) | [hsolbrig/template-fun-model](https://github.com/hsolbrig/template-fun-model) |
| [test-linkml-cookiecutter](https://yibeichan.github.io/test-linkml-cookiecutter/)<br/>test linkml cookiecutter |  | 0 [(details)](details/yibeichan_test-linkml-cookiecutter.md) | [yibeichan/test-linkml-cookiecutter](https://github.com/yibeichan/test-linkml-cookiecutter) |
| [tkg-beacon](https://ncats-tangerine.github.io/tkg-beacon/)<br/>Knowledge Beacon wrapper of a Translator Knowledge Graph data model compliant Neo4j Database |  | 0 [(details)](details/NCATS-Tangerine_tkg-beacon.md) | [NCATS-Tangerine/tkg-beacon](https://github.com/NCATS-Tangerine/tkg-beacon) |
| [tmpl-cm](https://netbeheer-nederland.github.io/tmpl-cm/)<br/>Template repository for conceptual models. |  | 0 [(details)](details/Netbeheer-Nederland_tmpl-cm.md) | [Netbeheer-Nederland/tmpl-cm](https://github.com/Netbeheer-Nederland/tmpl-cm) |
| [transient-heat-transfer-analysis-model](https://multiplanetaryhomestead.github.io/transient-heat-transfer-analysis-model/)<br/>Demo m30ml project for conducting a transient heat transfer analysis |  | 0 [(details)](details/multiplanetaryhomestead_transient-heat-transfer-analysis-model.md) | [multiplanetaryhomestead/transient-heat-transfer-analysis-model](https://github.com/multiplanetaryhomestead/transient-heat-transfer-analysis-model) |
| [vrs_linkml](https://sierra-moxon.github.io/vrs_linkml/)<br/>LinkML Version of the GA4GH Variation Representation Specification |  | 0 [(details)](details/sierra-moxon_vrs_linkml.md) | [sierra-moxon/vrs_linkml](https://github.com/sierra-moxon/vrs_linkml) |
| [zfin-genotype-to-phenotype-ingest](https://monarch-initiative.github.io/zfin-genotype-to-phenotype-ingest/) |  | 0 [(details)](details/monarch-initiative_zfin-genotype-to-phenotype-ingest.md) | [monarch-initiative/zfin-genotype-to-phenotype-ingest](https://github.com/monarch-initiative/zfin-genotype-to-phenotype-ingest) |